### ISSUE TITLE
User can read full recipe in a larger view (Name, description, ingredients WITH quantity info, directions)

### ISSUE THREAD

**Issue by armsnyder** [eecs394-spr15/StockMyBar#58]

**Comment by missfxhr** [comment]
We now already have name, directions and description. So I think quantity is what we only need to implement.

**Comment by armsnyder** [comment]
Yes, but it might make sense for the user to be able to tap on a recipe and open it in a full-screen view? The text is pretty small and squeezed into a very small view.

**Comment by missfxhr** [comment]
oh I got u, that makes sense!

**Comment by armsnyder** [comment]
This was not completed. There is still no quantity info in the ingredients list.

**Comment by armsnyder** [comment]
We want the recipe to also show something like this:
- 1 cl brandy
- 2 cl Cointreau
- 2 cl lemon juice
